### PR TITLE
Update plugins formatting & ingress to v1

### PR DESCRIPTION
**Pull request by fmuyassarov**
Remove some duplication from `args`, update the plugins.yaml formatting as suggested [here](https://github.com/kubernetes/test-infra/issues/20631#issuecomment-787693609), update the ingress api to v1.

### CODE REVIEW

File: config/prow/cluster/starter-s3.yaml [hunk 1]
```diff
@@ -532,7 +532,7 @@ spec:
   - port: 80
     targetPort: 8888
 ---
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```
Comment (fejta) [inline config/prow/cluster/starter-s3.yaml:535]: I suspect this will break kubectl apply, right?

**Review by fejta — APPROVED** [review]
/hold

File: config/prow/cluster/starter-s3.yaml (hunk 1 above)
Comment (fmuyassarov) [reply config/prow/cluster/starter-s3.yaml:535]: Do you mean because there might be `extensions/v1beta1` version still available on the cluster?

Comment (fejta) [reply config/prow/cluster/starter-s3.yaml:535]: Oh sorry these are both just starter examples? Nevermind.
/hold cancel

**Review by fejta — APPROVED** [review]
/ok-to-test

File: config/prow/cluster/starter-s3.yaml (hunk 1 above)
Comment (fejta) [reply config/prow/cluster/starter-s3.yaml:535]: (my concern is that changing the api version of our production ingress resources will cause our production deployments to start failing as I would expect that field to be immutable (could be wrong, but not something I want to merge until the morning)

Comment (fmuyassarov) [reply config/prow/cluster/starter-s3.yaml:535]: Yes I agree, I was hesitating too to update it, but went ahead considering these are starters for someone like me who is trying to setup a prow cluster.

Comment (fejta) [reply config/prow/cluster/starter-s3.yaml:535]: Totally, thanks so much for cleaning this up!!

**Pull request merged**